django_freeradius/utils.py

```diff
@@ -72,9 +72,9 @@ def prefix_generate_users(prefix, n, password_length):
 
 def generate_pdf(prefix, data):
     template = get_template(BATCH_PDF_TEMPLATE)
-    html = template.render(data)
-    f = open("{}/{}.pdf".format(settings.MEDIA_ROOT, prefix), "w+b")
-    pisa.CreatePDF(html.encode("utf-8"), dest=f, encoding="utf-8")
+    html = HTML(string=template.render(data), base_url=BATCH_PDF_TEMPLATE_BASE_URL)
+    f = open("{}/{}.pdf".format(settings.MEDIA_ROOT, prefix), "wb+")
```

Comment:
Is this change necessary?

Reply:
No, will turn it back, thanks for the comment